[2022-12-08 16:25:12,676] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: move_data_from_s3_to_redshidt_with_awswrangler.move_data_from_s3_to_redshift scheduled__2022-12-07T00:00:00+00:00 [queued]>
[2022-12-08 16:25:12,682] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: move_data_from_s3_to_redshidt_with_awswrangler.move_data_from_s3_to_redshift scheduled__2022-12-07T00:00:00+00:00 [queued]>
[2022-12-08 16:25:12,682] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-08 16:25:12,682] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-12-08 16:25:12,682] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-08 16:25:12,692] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): move_data_from_s3_to_redshift> on 2022-12-07 00:00:00+00:00
[2022-12-08 16:25:12,697] {standard_task_runner.py:52} INFO - Started process 555 to run task
[2022-12-08 16:25:12,699] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'move_data_from_s3_to_redshidt_with_awswrangler', 'move_data_from_s3_to_redshift', 'scheduled__2022-12-07T00:00:00+00:00', '--job-id', '95', '--raw', '--subdir', 'DAGS_FOLDER/move_data_s3redshift.py', '--cfg-path', '/tmp/tmpvzzcu5u1', '--error-file', '/tmp/tmpxp2yzv75']
[2022-12-08 16:25:12,700] {standard_task_runner.py:80} INFO - Job 95: Subtask move_data_from_s3_to_redshift
[2022-12-08 16:25:12,710] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:470: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2022-12-08 16:25:12,737] {task_command.py:369} INFO - Running <TaskInstance: move_data_from_s3_to_redshidt_with_awswrangler.move_data_from_s3_to_redshift scheduled__2022-12-07T00:00:00+00:00 [running]> on host 8458a2c1ba2e
[2022-12-08 16:25:12,791] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=move_data_from_s3_to_redshidt_with_awswrangler
AIRFLOW_CTX_TASK_ID=move_data_from_s3_to_redshift
AIRFLOW_CTX_EXECUTION_DATE=2022-12-07T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-07T00:00:00+00:00
[2022-12-08 16:25:12,803] {credentials.py:1120} INFO - Found credentials in environment variables.
[2022-12-08 16:25:12,895] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/move_data_s3redshift.py", line 21, in read_from_s3_and_insert_redshift
    s3_data = wr.s3.read_parquet(path=s3_path)
  File "/home/airflow/.local/lib/python3.7/site-packages/awswrangler/s3/_read_parquet.py", line 751, in read_parquet
    raise exceptions.NoFilesFound(f"No files Found on: {path}.")
awswrangler.exceptions.NoFilesFound: No files Found on: s3://***-jkimera-bucket/from_red.parquet.
[2022-12-08 16:25:12,902] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=move_data_from_s3_to_redshidt_with_awswrangler, task_id=move_data_from_s3_to_redshift, execution_date=20221207T000000, start_date=20221208T162512, end_date=20221208T162512
[2022-12-08 16:25:12,911] {standard_task_runner.py:97} ERROR - Failed to execute job 95 for task move_data_from_s3_to_redshift (No files Found on: s3://***-jkimera-bucket/from_red.parquet.; 555)
[2022-12-08 16:25:12,951] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-12-08 16:25:12,981] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
